src/attributes.adoc

```diff
@@ -67,6 +67,8 @@ endif::[]
 :TAG_RESET_CSR: The tag of the CSR must be reset to zero. The reset values of the metadata and address fields are UNSPECIFIED.
 :REQUIRE_CRE_CSR: Access to this CSR is illegal if CRE for the current mode is zero (see <<section_cheri_disable>>).
 
+:CAP_MODE: 0
+:INT_MODE: 1
```

Comment:
```suggestion
:CAP_MODE_VALUE: 0
:INT_MODE_VALUE: 1
```

Maybe something like this to indicate the macro holds the integer M bit value for the mode?

Reply:
I've done a find-and-replace on all the uses, so will commit that instead